todo_app/data/trello_items.py

```diff
@@ -0,0 +1,59 @@
+import requests
+import os
+
+
+class Item:
+    id = ''
+    status = ''
+    name = ''
+
+    def __init__(self, id, status, name):
+        self.id = id
+        self.status = status
+        self.name = name
+
+
+def get_items():
+    json = get_open_cards()
+    items = []
+
+    for list in json:
+        list_name = list['name']
+        for card in list['cards']:
+            items.append(Item(card['id'],list_name,card['name']))
+
+    return items
+
+
+def add_item(name):
+    list_id = get_list_id('To Do')
+    requests.post('https://api.trello.com/1/cards', params={'key':os.getenv('API_KEY'),'token':os.getenv('API_TOKEN'),'idList':list_id,'name':name})
+
+
+def get_open_cards():
+    r = requests.get('https://api.trello.com/1/boards/' + os.getenv('TRELLO_BOARD_ID') + '/lists', params={'key':os.getenv('API_KEY'),'token':os.getenv('API_TOKEN'),'cards':'open'})
+    return r.json()
+
+
+def move_to_do(id):
+    list_id = get_list_id('To Do')
+    requests.put('https://api.trello.com/1/cards/' + id, params={'key':os.getenv('API_KEY'),'token':os.getenv('API_TOKEN'),'idList':list_id})
```

Comment:
Since I can't find anything else to comment on, so I'll be pedantic - this is vulnerable to injection via the `id` parameter e.g.

```sh
curl -X 'POST' 'http://localhost:5000/doing/62725927c94afd3d34d82e94%3Fname%3Dinjected_name_change'
```

You could fix this by encoding the `id` first

```python
import urllib.parse
encoded_id = urllib.parse.quote(id, safe='')
```

Reply:
Id is now encoded